Language/Generics/Superbounded_types/well_bounded_A01_t03.dart

```diff
@@ -5,23 +5,30 @@
 /// @assertion Any use of a type `T` which is not well-bounded is a
 /// compile-time error.
 ///
-/// @description Checks that it is a compile-time error when a class with a not
-/// well-bounded type parameter is declared
+/// @description Checks that it is a compile-time error when a class has a type
+/// parameter bound which is a raw type that does not have simple bounds, and
+/// when a class has a type parameter bound which is malbounded.
 /// @author iarkh@unipro.ru
 
 class A<T extends A<T>> {}
 
+// Here B1 is a well-bounded type but the bound is a raw type A<A<dynamic>> that
+// doesn't have simple bounds
```

Comment:
That's great! A few more words could help a future reader, just mentioning it the first time we encounter this situation:
```suggestion
// doesn't have simple bounds, and that's a compile-time error.
```